libs/core-wrappers/src/lib/Navbar/Navbar.tsx

```diff
@@ -18,7 +18,7 @@ export const NavItem: React.FC<
           isActive ? styles['nav-active'] : ''
         }`}
       >
-        {icon && <Icon name={icon} className={styles['nav-icon']} />}
+        {icon && <Icon name={icon} size="xs" className={styles['nav-icon']} />}
```

Comment:
**Note:** It was already a small icon 😅. This actually makes it bigger. It sets size explicitly (to match design) instead of using font-size. This is an optional parameter (cuz maybe sometimes the icon should match text size; it happens in CMS).